### ISSUE TITLE
No div id="content" for skipnav #content in layout.phtml

### ISSUE THREAD

**Issue by mdavismcd** [omeka-s-themes/papers#14]
I may be missing something because the code is slightly different compared with the default theme and therefore function differently, but the skip to main content doesn't work on my Omeka S sites unless I go and add `<div id="content" role="main">` in the page-content main class section:

Original:
```
<main class="page-content" aria-label="<?php echo $this->escapeHtml($this->translate('Content')); ?>">
            <div class="wrapper">
            <?php echo $this->content; ?>
            </div>
```

Updated with div id:
```
<main class="page-content" aria-label="<?php echo $this->escapeHtml($this->translate('Content')); ?>">
            <div class="wrapper">
            <div id="content" role="main">
            <?php echo $this->content; ?>
            </div>
```

Thanks for any clarification.

**Comment by kimisgold** [comment]
Sorry to have missed this! We've updated the markup to now include the content id needed. [The fix should be in v1.3.2.](https://github.com/omeka-s-themes/papers/releases/tag/v1.3.2)

**Comment by mdavismcd** [comment]
Thanks so much, @kimisgold !!!
